core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -720,8 +712,8 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
       }
 
       withClue("Text in the pubsub error message was incorrect:") {
-        errorMsg.attributes("errorMessage") shouldBe
-          "All entities failed to update. There were 1 errors in total. Error messages: test-type this-entity-does-not-exist not found"
+        errorMsg.attributes("errorMessage") should contain
+        "All entities failed to update. There were 1 errors in total."
```

Comment:
Quicksilver doesn't report exactly which reference isn't found, and I'm ok with that. It's not a huge risk but NOT echoing the user input is a good security stance to protect against XSS, etc.